docs/overview.rst

```diff
@@ -1,80 +0,0 @@
-********
-Overview
```

Comment:
Is this information going to get ported to the web page?  I think a lot of it is fairly critical and needs to be somewhere fairly easy-to-find (but the web page is definitely that)